tools/disclosure-submitted/generate-diff.js

```diff
@@ -7,39 +7,45 @@ const octokit = new Octokit();
 
 const PR_NUMBER = process.env.PR_NUMBER.toString();
 
-const files = await octokit.pulls.listFiles({
+const files = await octokit.pulls
+  .listFiles({
     owner: "418sec",
     repo: "huntr",
-    pull_number: PR_NUMBER
-})
-.then(response => { return response.data })
-.catch(() => { return "N/A" })
-
-if(files == "N/A") {
-    core.setFailed("Error whilst getting list of files...")
+    pull_number: PR_NUMBER,
+  })
+  .then((response) => {
+    return response.data;
+  })
+  .catch(() => {
+    return "N/A";
+  });
+
+if (files == "N/A") {
+  core.setFailed("Error whilst getting list of files...");
 }
 
-if(files.length === 0) {
-    core.setFailed("No difference between HEAD and BASE...")
+if (files.length === 0) {
+  core.setFailed("No difference between HEAD and BASE...");
 }
 
-let diff = []
-
-files.forEach(file => {
-    let change = 
-        file.status === "added" ? "A" : 
-        file.status === "removed" ? "D" : 
-        file.status === "modified" ? "M" : 
-        "Error"
-
-    diff.push({
-        change: change,
-        path: file.filename
-    });
-})
+let diff = [];
 
-console.log(`The diff generated by this PR ${PR_NUMBER} is: \n`, diff)
+files.forEach((file) => {
```

Comment:
@JamieSlome - I'm unsure as to where these additions were made from, they weren't from myself. Let me know if you would like it included or not and I can have it modified, ty x

Reply:
@benharvie - I would say restore these to the current versions on staging - same for the `package-lock.json`.

Reply:
package-lock has been done, the changes on generate-diff are now only linting changes when compared to `staging`, cheers 👻